source/GlassView.Core.Test/Convenience.cs

```diff
@@ -0,0 +1,11 @@
+using System.Text.Json;
+
+namespace Atmoos.GlassView.Core.Test;
+
+public static class Convenience
+{
+    private static readonly JsonSerializerOptions options = new JsonSerializerOptions() { WriteIndented = true }.EnableGlassView();
+
+    public static String Serialize<T>(this T value) => JsonSerializer.Serialize(value, options);
```

Comment:
I'm not sure if hiding the options behind this convenience method, brings any value to the codebase. What do you think?
Do you think we would have some benefits from seeing in 6 months from now that the serialization uses options?

Reply:
There is no intention to hide the options. These `Deserialize` and `Serialize` methods are pure convenience for when the options are irrelevant for what's being tested.

They are only used in three test cases within one test class ( and only in this project) where these options are IMHO not relevant. (Hence also the ignore whitespace and new line parameterisation in the respective tests.)

However, where they are relevant they are exposed publicly.  (in the `DirectoryExport` where they are taken as constructor parameters and in the `GlassView` static class where they are "actively" configured by mapping the appsettings to the json sttings).

As of now, no GlassView user (person) will need to touch the json settings directly (as IMHO they shouldn't really care how they "look" like or even care about the fact that we have chosen to use json to deserialize our models)

Reply:
To me this is also very much a topic that is very strongly related to a "test strategy". So, here too, I hope it's ok for you if we discuss this thoroughly in the context of issue #14.

I hope that's ok for you for now?

Reply:
I've attempted to make this a little clearer in the tests and also the Convenience class. Also I hope to have now chosen more descriptive names for the test cases, along with quite a bit of reasoning.
Please see the test: [SerializationTest.cs](https://github.com/atmoos/GlassView/blob/exportBenchmarkResults/source/GlassView.Core.Test/Serialization/SerializationTest.cs)

Reply:
In the test classes we can do whatever we please. This means we can also have convenience classes.